src/main/java/mate/academy/internet/shop/controller/DeleteOrderController.java

```diff
@@ -1,22 +1,21 @@
-package mate.academy.internetshop.controller;
+package mate.academy.internet.shop.controller;
```

Comment:
1)Let's change package naming(in pom.xml) from mate.academy to something more specific to make your project more individual;